user-guide/Getting_started/Installing_a_DMA/Installing_DM_using_the_DM_Installer.md

```diff
@@ -70,7 +70,7 @@ If you are using DataMiner Installer v10.4, follow the steps below to install Da
    - *Organization API Key*: Provide an organization key that has the necessary permissions to add DataMiner nodes in your organization. For more information on how you can add a new organization key to your organization on dataminer.services, see [Managing dataminer.services keys](xref:Managing_DCP_keys).
    - *System Name*: This name will be used to identify the DataMiner System in various dataminer.services applications.
    - *System URL*: This URL will grant you remote access to your DataMiner System web applications. You can choose to either [disable or enable this remote access feature](xref:Controlling_remote_access) at any time.
-   - *Admin Email*: This email is associated with the dataminer.services account that has the administrator role for the organization.
+   - *Admin Email*: This email is associated with the dataminer.services account that is part of the organization of which the key is used and will become owner of the DMS on dataminer.services.
```

Comment:
This is rather confusing to read. Since we've already mentioned next to "Organization API Key" that it's "your organization" (meaning, the organization the reader is a member of), I think it might be clearer if we just used that terminology instead (same for the other file):
```suggestion
   - *Admin Email*: This email address must be associated with a dataminer.services account that is a member of your organization. It will become the owner of the DMS on dataminer.services.
```